app/views/responsible_body/devices/schools/show.html.erb

```diff
@@ -7,23 +7,29 @@
       <%= @school.name %>
     </h1>
 
-    <%- if @school.has_std_device_allocation? %>
-      <%= render DeviceCountComponent.new(
-        school: @school,
-        show_action: @school.std_device_allocation.devices_ordered < @school.std_device_allocation.cap,
-        action: { 'Order devices' => order_devices_responsible_body_devices_school_path(urn: @school.urn) }) %>
-    <% else %>
-      <%= render partial: 'shared/school_without_allocation' %>
-    <%- end %>
+    <%- if FeatureFlag.active?(:virtual_caps) && @responsible_body.vcap_feature_flag %>
```

Comment:
I added `ResponsibleBody#has_virtual_cap_feature_flags?` that does this late yesterday. Sorry, should have let everyone know.

Reply:
done :+1: